test/remote/gateways/remote_visanet_peru_test.rb

```diff
@@ -107,7 +107,7 @@ def test_successful_refund_unsettled
     assert_success response
 
     new_auth = "_|#{response.authorization.split('|')[1]}"
-    refund = @gateway.refund(@amount, new_auth, @options.merge(force_full_refund_if_unsettled: true, ruc: '20341198217'))
+    @gateway.refund(@amount, new_auth, @options.merge(force_full_refund_if_unsettled: true, ruc: '20341198217'))
```

Comment:
This should probably be `_refund`, or just comment out the entire method. I'm favoring the latter.